src/bluehawk/commands/EmphasizeCommand.ts

```diff
@@ -0,0 +1,21 @@
+import {
+  makeBlockOrLineCommand,
+  NoAttributes,
+  NoAttributesSchema,
+} from "./Command";
+
+import { removeMetaRange } from "./removeMetaRange";
+
+export const EmphasizeCommand = makeBlockOrLineCommand<NoAttributes>({
+  attributesSchema: NoAttributesSchema,
+
+  process({ commandNode, parseResult }) {
+    const { source } = parseResult;
+    const { text } = source;
+
+    // Strip tags
+    removeMetaRange(text, commandNode);
+    commandNode.attributes = {};
```

Comment:
Actually, you'll want to set the file attributes on the `source` object, not the command node.

Also, be sure not to clobber any existing attributes (as is happening here - what if commandNodes already had an attributes list?)